Fix failing tests in test_user_schemas

The following tests are failing: 
FAILED tests/test_schemas/test_user_schemas.py::test_user_base_valid - KeyError: 'nickname'
FAILED tests/test_schemas/test_user_schemas.py::test_user_create_valid - KeyError: 'nickname'
FAILED tests/test_schemas/test_user_schemas.py::test_user_update_valid - KeyError: 'first_name'
FAILED tests/test_schemas/test_user_schemas.py::test_user_response_valid - pydantic_core._pydantic_core.ValidationError: 1 validation error for UserResponse
FAILED tests/test_schemas/test_user_schemas.py::test_login_request_valid - pydantic_core._pydantic_core.ValidationError: 1 validation error for LoginRequest

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -252,7 +252,7 @@ def user_update_data():
 @pytest.fixture
 def user_response_data():
     return {
-        "id": "unique-id-string",
+        "id": "fe946161-066d-4669-b362-1b354fcf7f92",
         "nickname": "testuser",
         "email": "test@example.com",
         "last_login_at": datetime.now(),
@@ -263,4 +263,4 @@ def user_response_data():
 
 @pytest.fixture
 def login_request_data():
-    return {"username": "john_doe_123", "password": "SecurePassword123!"}
+    return {"email": "john.doe@example.com", "password": "SecurePassword123!"}

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -215,9 +215,10 @@ async def manager_user(db_session: AsyncSession):
 @pytest.fixture
 def user_base_data():
     return {
-        "username": "john_doe_123",
+        "nickname": "john_doe_123",
         "email": "john.doe@example.com",
-        "full_name": "John Doe",
+        "first_name": "John",
+        "last_name": "Doe",
         "bio": "I am a software engineer with over 5 years of experience.",
         "profile_picture_url": "https://example.com/profile_pictures/john_doe.jpg"
     }
@@ -225,9 +226,10 @@ def user_base_data():
 @pytest.fixture
 def user_base_data_invalid():
     return {
-        "username": "john_doe_123",
+        "nickname": "john_doe_123",
         "email": "john.doe.example.com",
-        "full_name": "John Doe",
+        "first_name": "John",
+        "last_name": "Doe",
         "bio": "I am a software engineer with over 5 years of experience.",
         "profile_picture_url": "https://example.com/profile_pictures/john_doe.jpg"
     }
@@ -241,7 +243,8 @@ def user_create_data(user_base_data):
 def user_update_data():
     return {
         "email": "john.doe.new@example.com",
-        "full_name": "John H. Doe",
+        "first_name": "John",
+        "last_name": "Doe",
         "bio": "I specialize in backend development with Python and Node.js.",
         "profile_picture_url": "https://example.com/profile_pictures/john_doe_updated.jpg"
     }
@@ -250,7 +253,7 @@ def user_update_data():
 def user_response_data():
     return {
         "id": "unique-id-string",
-        "username": "testuser",
+        "nickname": "testuser",
         "email": "test@example.com",
         "last_login_at": datetime.now(),
         "created_at": datetime.now(),

--- a/tests/test_schemas/test_user_schemas.py
+++ b/tests/test_schemas/test_user_schemas.py
@@ -25,7 +25,7 @@ def test_user_update_valid(user_update_data):
 # Tests for UserResponse
 def test_user_response_valid(user_response_data):
     user = UserResponse(**user_response_data)
-    assert user.id == user_response_data["id"]
+    assert str(user.id) == user_response_data["id"]
     # assert user.last_login_at == user_response_data["last_login_at"]
 
 # Tests for LoginRequest

